core/src/com/mygdx/game/character/Character.java

```diff
@@ -115,4 +135,13 @@ public void dispose() {
         textureAtlas.dispose();
         bitmapFont.dispose();
     }
+
+    public void applyDamage(final int damage) {
+        int actualDamage = damage - characterConfig.getCharacterStats().getDefense()/2;
```

Comment:
Will the damage be always /2?

Reply:
No, it depends on the defense of the character.
 `Actual damage = Damage - (Defense/2)`  
So this is basically making the defense value a damage reduction parameter. Another way to put this would be
`Damage reduction = 0.5 x Defense` 
We might want to have pickups or spells later on which can change increase/decrease attack, defense etc. This will work in conjunction with those. We can experiment with the multiplier but with the current values we have set, this seemed to work well.

Reply:
That makes sense. We will add the pickups or spells later. Works now